WordPress/Classes/Extensions/NSFetchedResultsController+Helpers.swift

```diff
@@ -34,4 +34,20 @@ extension NSFetchedResultsController {
 
         return object(at: indexPath) as? NSManagedObject
     }
+
+    /// Returns whether an NSFetchedResultsController is empty
+    ///
+    @objc func isEmpty() -> Bool {
+        // We can not return fetchedObjects.count == 0 because of a Swift compiler error:
+        // Extension of a generic Objective-C class cannot access the class's generic parameters at runtime
+        guard let sections = sections else {
+            return false
+        }
+
+        let initialCount = 0
+
+        return sections.reduce(initialCount, { result, section in
```

Comment:
This one took me a while to understand, so I pinged @jleandroperez for a suggestion and this is what he came up with:

```
func isEmpty() -> Bool {
     return sections?.first(where: { $0.numberOfObjects > 0 }) == nil
}
```

I really like it, what do you think?

Reply:
I've just pushed an update to use it.